src/host/exe/exemain.cpp

```diff
@@ -0,0 +1,334 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
ugh, nobody can tell that this file was renamed so it's going to become a real pain to un-merge-conflict when Dragos' changes come back out. It's also hard to review.

This is not a complaint about your work. This is the mechanics of git!

Reply:
for easier reviewing, check out this branch and...

```
git diff origin/main:src/host/exemain.cpp HEAD:src/host/exe/exemain.cpp
```